assets/src/scss/layout/_country-selector.scss

```diff
@@ -0,0 +1,195 @@
+
+#country-selector {
+  --country-selector-- {
+    color: $white;
+    font-size: $font-size-sm;
+  }
+  margin-bottom: 24px;
+  text-align: start;
+  width: 100%;
+
+  a --country-selector--a-- {
+    color: $white;
+  }
+
+  a:hover,
+  button:hover {
+    text-decoration: var(--link--hover--text-decoration, underline);
+  }
+
+  button:focus {
+    border-radius: var(--link--focus--border-radius, 4px);
+    outline: var(--link--focus--border, 2px solid rgba(255, 255, 255, 0.4));
+    outline-offset: 0;
+  }
+
+  button:focus:not(:focus-visible) {
+    outline: 0;
+  }
+
+  .country-selector-toggle-container {
+    --country-selector--toggle-container-- {
+      border-bottom: 1px solid rgba(255, 255, 255, 0.5);
+    }
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding-bottom: 24px;
+    transition: border 0s linear 0.5s;
+    vertical-align: middle;
+    width: 100%;
+  }
+
+  .control-toggle {
+    align-self: flex-start;
+    background: transparent;
+    border: none;
+    color: var(--site-footer--color, $white);
+    cursor: pointer;
+    text-align: start;
+    white-space: nowrap;
+    width: 100%;
+
+    &::before {
```

Comment:
`&::before` and `&::after` are sharing some attributes that you can put them together like this

```
&::before, 
&::after {
background-color: var(--site-footer--color, $white);
content: "";
display: inline-block;
width: 16px;
}
```